tests/FSharp.CodeFormat.Tests/Tests.fs

```diff
@@ -44,6 +44,65 @@ let ``Simple code snippet is formatted with tool tips``() =
     | _ -> false)
   |> shouldEqual true
 
+[<Test>]
+let ``Preview language feature from FSharp Core is supported``() =
+    let source = """
+let x = 12
+nameof x
+"""
+    let snips, errors = agent.ParseSource("/somewhere/test.fsx", source.Trim())
+
+    errors |> shouldEqual [||]
+    snips |> containsSpan (function
+        | Token(_, "nameof", Some (ToolTipWithLiteral "val nameof : 'T -> string")) -> true
+        | _ -> false)
+    |> shouldEqual true
+
+[<Test>]
+let ``Preview language feature is supported``() =
```

Comment:
I'll pedantically remark that this test will stop to serve its purpose once the particular feature set gets released. Perhaps add a comment noting that for posterity?